payment-circle/src/main/java/org/stellar/anchor/paymentservice/circle/model/CircleWallet.java

```diff
@@ -16,7 +17,11 @@
     List<CircleBalance> balances;
 
     public Account.Capabilities getCapabilities() {
-        return new Account.Capabilities(Network.CIRCLE, Network.STELLAR);
+        List<Network> sendOrReceiveCapabilities = new ArrayList<>(List.of(Network.CIRCLE, Network.STELLAR));
+        if ("merchant".equals(type)) {
+            sendOrReceiveCapabilities.add(Network.BANK_WIRE);
+        }
+        return new Account.Capabilities(sendOrReceiveCapabilities.toArray(new Network[0]));
```

Comment:
❓ What is this last line doing? It seems to me like you should just `return sendOrReceiveCapabilities;`

Reply:
> It seems to me like you should just `return sendOrReceiveCapabilities;`

Not exactly, sendOrReceiveCapabilities is a list but I need to return an `Account.Capabilities` object.

You're right though regarding this chunk of code being a bit weird. Improved that on https://github.com/stellar/java-stellar-anchor-sdk/pull/21/commits/51f583f618786cbe534f8404b33b14af4cc8ec32.